changelog/unreleased/kong/fix-grpc-gateway-json-decode-bug.yml

```diff
@@ -0,0 +1,3 @@
+message: "**grpc-gateway**:Fixed an json decode issue so that kong will response 400 with error info in it's body instead of 500."
```

Comment:
```suggestion
message: "**grpc-gateway**: When there is a JSON decoding error, respond with status 400 and error information in the body instead of status 500."
```

Reply:
Sorry for my poor English.😂